climakitae/indices.py

```diff
@@ -0,0 +1,93 @@
+"""Functions for deriving fire indices"""
+
+import xarray as xr
+
+
+def fosberg_fire_index(t2_F, rh_percent, windspeed_mph):
+    """Compute the Fosberg Fire Weather Index.
+    Use hourly weather as inputs.
+    Ensure that the input variables are in the correct units (see below).
+
+    Parameters
+    ----------
+    t2_F: xr.DataArray
+        Air temperature in units of Fahrenheit
+    rh_percent: xr.DataArray
+        Relative humidity in units of 0-100 (percent)
+    windspeed_mph: xr.DataArray
+        Windspeed in units of miles per hour
+
+    Returns
+    -------
+    xr.DataArray
+        Fosberg Fire Weather Index computed for each grid cell
+
+    References
+    ----------
+    https://a.atmos.washington.edu/wrfrt/descript/definitions/fosbergindex.html
+    https://github.com/sharppy/SHARPpy/blob/main/sharppy/sharptab/fire.py
+    https://www.spc.noaa.gov/exper/firecomp/INFO/fosbinfo.html
+
+    """
+    # Compute the equilibrium moisture constant
+    m_low, m_mid, m_high = _equilibirum_moisture_constant(h=rh_percent, T=t2_F)
+
+    # For RH < 10%, use the low m value.
+    # For RH >= 10%, use the mid value
+    m = xr.where(rh_percent < 10, m_low, m_mid)
+    # For RH > 50%, use the high m value.
+    m = xr.where(rh_percent > 50, m_high, m)
+
+    # Compute the moisture dampening coefficient
+    n = _moisture_dampening_coeff(m)
+
+    # Compute the index
+    U = windspeed_mph
+    FFWI = (n * ((1 + U**2) ** 0.5)) / 0.3002
```

Comment:
Does there need to be an additional line here (or in _convert_units) restricting the maximum possible output to be 100? Currently if I force values of 100F, 0% humidity, and 50mph windspeed, it returns a FFWI value of ~166 

From the a.atmos.washington.edu reference: "The values were designed so that a index rating of 100 is equal to a moisture content of 0 and a wind speed of 30 mph. Larger combinations of these values still result in an index of 100."

Reply:
great catch, I'll fix this!